dist/enhanced-shutter-card.js

```diff
@@ -297,18 +312,24 @@ const SHUTTER_CSS =`
         margin: auto;
         width: 100%;
         height: 100%;
+        
+        image-rendering: auto;
+        image-rendering: pixelated;
+        image-rendering: crisp-edges;
       }
       .${ESC_CLASS_SELECTOR_SLIDE} {
         z-index: -1;
         position: absolute;
         background-position: bottom;
         overflow: hidden;
         top: 0;
-        width: 100%;
+        
```

Comment:
Why 3 rowes with image-rendering?

Reply:
Due to different Browser compatibility.
https://stackoverflow.com/questions/68770680/image-rendering-crisp-edges-and-smooth-values-report-invalid-property-value-in

This way the browser picks the first supported value.